### DISCUSSION TITLE
Minify / Optimization of _site directory post rendering ?

### DISCUSSION THREAD

**Discussion by bugsbunny88** [quarto-dev/quarto-cli#6511] [Q&A]
### Description

Hello team,
great project. been using it for a while. 
But was wondering if any of you have utilized techniques or tools to further optimize _site directory quarto renders . Minification / Optimization to further optimize the site loading times etc. And perhaps share the process etc. ?

**Comment by mcanouil** [comment]
Far as I know, there is no optimisation attempts that are not part of Quarto development itself.

**Comment by jimjam-slam** [comment]
You could always write a TypeScript post-render script to handle that minification!
